### ISSUE TITLE
Behavior of pattern/@documents

### ISSUE THREAD

**Issue by xatapult** [schxslt/schxslt#209]
The ISO skeleton implementation resolves a `pattern/@documents` value as relative the source XML document's location.

SchXslt resolves this as relative to the *schema* location. 

The standard leaves this undefined. However, I think the ISO interpretation makes more sense: When a source document contains a relative reference to some other document, this path is usually relative to the source document. Resolving it relative to the schema document (which could be anywhere) does IMHO not make much sense.

**Comment by dmj** [comment]
Yes, resolving the URIs relative to the source document makes sense. To be exact: Currently SchXslt resolves the URIs relative to the validation stylesheets location.
